deepspeed/pt/deepspeed_cuda.py

```diff
@@ -393,21 +401,24 @@ def backward(ctx, grad_output):
 class DeepSpeedTransformerLayer(nn.Module):
     """Initialize the DeepSpeed Transformer Layer.
 
+        Static variable:
+            layer_id: The layer-index counter starting from 0 and incrementing by 1 every time a layer object is instantiated,
```

Comment:
Will users need to use `layer_id`? If it's just an implementation detail, this might be better as a comment instead of the public docstring.

Reply:
@RezaYazdaniAminabadi thoughts here?

Reply:
Yes, I agree with Shaden. That can just a comment. We can remove the detail from public docstring.